src/main/java/wco/datamodel/wco/res_dms/_2/Response.java

```diff
@@ -360,8 +360,8 @@
      * <p>
      * You are getting this "catch-all" property because of the following reason: 
      * The field name "Declaration" is used by two different parts of a schema. See: 
-     * line 1426 of file:/Users/developer/workspace/hmrc/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
-     * line 832 of file:/Users/developer/workspace/hmrc/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
+     * line 1426 of file:/home/pmonteiro-work/Workspace/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
+     * line 832 of file:/home/pmonteiro-work/Workspace/wco-dec/updateSchema/customs-declarations/public/api/conf/1.0/schemas/wco/notification/WCO_RES_2_DMS.xsd
```

Comment:
Nit: It's probably automaticaly generated but relative path should be enough. Not to expose your own filesystem.

Reply:
Ahhh good point. Let me remove that.
Done.